modules/microshift-greenboot-workload-health-script-ex.adoc

```diff
@@ -0,0 +1,55 @@
+/Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-greenboot-workload-scripts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-greenboot-workload-health-check-script-ex_{context}"]
+= Workload health check script example
+
+The following example uses the {microshift-short} health check script as a template. You can use this example with the provided libraries as a guide for creating basic health check scripts for your applications.
+
+[id="microshift-greenboot-app-health-check-basic-prereqs_{context}"]
+== Basic prerequisites for creating a health check script
+
+* The workload must be installed.
+* You must have root access.
+
+[id="microshift-greenboot-app-health-check-ex_{context}"]
+== Example and functional requirements
+
+You can start with the following example health check script. Modify it for your use case. In your workload health check script, you must complete the following minimum steps:
+
+* Set the environment variables.
+* Define the user workload namespaces.
+* List the expected pod count.
+
+[IMPORTANT]
+====
+Choose a name prefix for your application that ensures it runs after the `40_microshift_running_check.sh` script, which implements the {microshift-short} health check procedure for its core services.
+====
+
+.Example workload health check script
+[source, bash]
+----
+# #!/bin/bash
+set -e
+
+SCRIPT_NAME=$(basename $0)
+
+# Set greenboot to read and execute the workload health check functions library.
+source /usr/share/microshift/functions/greenboot.sh
+#Q: should this still be in the script if we deprecated it? see https://github.com/openshift/microshift/pull/4437/files#diff-01887b3e92e76300a404b954081c1b6c06f7417e7d74b8bdac0d16a071a56c5dR7
```

Comment:
The regular procedure is that we first deprecate something, before we remove it.
https://github.com/openshift/microshift/blob/main/packaging/greenboot/functions.sh#L16-L23

However, it turns out that we won't completely deprecate the script because it still contains parsing of the greenboot config file.
So we'll remove the functions that were related to checking health, but not those that are use to calculate the max duration/timeout.


I made a PR to make the message more precise: https://github.com/openshift/microshift/pull/4779

Reply:
Then it sounds like we should replace this example with an example of using the microshift-healthcheck command with options. Because we are no longer checking health here.